docs/user-documentation.rst

```diff
@@ -0,0 +1,425 @@
+.. _user-documentation:
+
+
+Installation
+=============
+
+Here is a step by step guide on how to install Switchboard. It will help you
+download Switchboard and set it up with your application.
+
+Install Switchboard and its the dependencies using ``pip``::
+
+    pip install switchboard
+
+Next we need to bootstrap Switchboard within your application. The best approach
+depends on which application framework you're using.
+
+Pyramid
+-------
+
+Switchboard has a pyramid add-on available to make pyramid setup easier::
+
+    pip install pyramid_switchboard
+
+Once the dependency is in place, there are several ways to make sure that
+``pyramid_switchboard`` is active and Switchboard is up and running. They are
+all equivalent.
+
+1. Add ``pyramid_switchboard`` to the ``pyramid.includes`` section of your
+   application's main configuration section::
+
+    [app:main]
+    ...
+    pyramid.includes = pyramid_switchboard
+
+2. Use the ``includeme`` function via `config.include`::
+
+    config.include('pyramid_switchboard')
+
+3. Optionally setup the ``switchboard.template_helpers.is_active`` helper
+   function. It can be used with the template engine to make it easier to
+   reference switches in the template. For example, if Jinja is being used,
+   the following lines in the ``production.ini`` will add the helper as a
+   test_::
+
+    jinja2.tests =
+        active = switchboard.template_helpers.is_active
+
+   This configuration makes checking switches very easy in Jinja templates::
```

Comment:
Hmm, maybe it's just how github's diff is doing syntax highlighting. It is considering this line to be part of the code.